lib/explorers/api-parameters.explorer.ts

```diff
@@ -130,6 +130,11 @@ export const exploreModelDefinition = (type, definitions) => {
     return type.name;
 };
 
+const getEnumType = (values: string[] | number[] | (string|number)[]): 'string' | 'number' => {
+    const hasString = values.filter(isString).length > 0;
```

Comment:
@fwoelffel This has an error, as it expects only array os strings:
`Cannot invoke an expression whose type lacks a call signature. Type '{ <S extends string>(callbackfn: (value: string, index: number, array: string[]) => value is S, t...' has no compatible call signatures`

Reply:
Setting it only the type to `(string|number)[]` solves it